### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by drewstone**

### CODE REVIEW

**Comment by drewstone** [comment]
Please make sure we improve this by EOD @shekohex @tbraun96

**Comment by shekohex** [comment]
Did a few updates here: #294

File: README.md [hunk 1]
```diff
@@ -18,39 +27,39 @@ Gadget is a comprehensive framework for building AVS services on Tangle and Eige
...
+### Installing the CLI
 
-1. Clone the repository:
+To install the Tangle CLI, run the following command:
 
 ```bash
-git clone https://github.com/webb-tools/gadget/
-git submodule update --init --recursive
-cd gadget
+cargo install --bin tangle-cli --git https://github.com/webb-tools/gadget --force
```
Comment (shekohex) [inline README.md:38]:
```suggestion
cargo install --bin cargo-tangle --git https://github.com/webb-tools/gadget --force
```

File: README.md [hunk 2]
```diff
@@ -6,7 +6,16 @@
 [![Rust Version](https://img.shields.io/badge/rust-1.74.0%2B-blue.svg)](https://www.rust-lang.org)
 
 # Gadget: A Powerful distributed AVS Framework
-Gadget is a comprehensive framework for building AVS services on Tangle and Eigenlayer. It provides a standardized framework for building task based systems and enables developers to clearly specify jobs, slashing reports, benchmarks, and tests for offchain and onchain service interactions. We plan to integrate with other restaking infrastructures over time, if you are a project that is interested please reach out!
+[Gadget SDK](./sdk) 
+| [CLI](./cli) 
+| [Tangle Operator Docs](https://docs.tangle.tools/operators/validator/introduction) 
+| [Tangle Developer Docs](https://foundry-rs.github.io/foundry)
```
Comment (Serial-ATA) [inline README.md:12]: This should be https://docs.tangle.tools/developers right?

File: blueprints/ecdsa-threshold-mpc/README.md [hunk 3]
```diff
@@ -1,57 +1,19 @@
...
-```sh
-cargo build
-```
-to build the project, and
+- and to build the gadget that uses the blueprint, run the following command:
 
-```sh
-cargo tangle gadget deploy
+```bash
+cargo build -p ecdsa-threshold-mpc --features=gadget
```
Comment (Serial-ATA) [inline blueprints/ecdsa-threshold-mpc/README.md:18]: There is no `gadget` feature?

File: sdk/README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+3. [Features](#features)
+4. [Examples](#examples)
+5. [Building and Deploying Blueprints](#building-and-deploying-blueprints)
+
+## Overview
+
+The Gadget SDK is a comprehensive toolkit for Gadget development, providing tools and libraries for 
+Blueprint development on both Tangle and EigenLayer. It offers a wide range of functionalities 
+including benchmarking, keystores, metrics, networking, logging, and process execution/management. 
+The SDK supports Native, WASM, and no-std contexts, making it useful across a variety of environments.
```
Comment (Serial-ATA) [inline sdk/README.md:18]: I think everything after the first sentence can be dropped, the features section already covers all of these things.

File: README.md [hunk 5]
```diff
@@ -18,39 +27,51 @@ Gadget is a comprehensive framework for building AVS services on Tangle and Eige
...
 ```bash
-git clone https://github.com/webb-tools/gadget/
-git submodule update --init --recursive
-cd gadget
+cargo tangle gadget create --name <blueprint_name>
 ```
-   
-2. Build the project:
+
+where `<blueprint_name>` is replaced with a custom name for the Blueprint/Gadget.
```
Comment (Serial-ATA) [inline README.md:49]: This clarification probably isn't needed.

**Review by Serial-ATA — COMMENTED** [review]
@Tjemmmic Some small comments

**Pull request merged**